spec/system/candidate_interface/continuous_applications/submitting/candidate_has_an_application_become_inactive_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.feature 'Candidate with submitted applications', :continuous_applications do
+  include CandidateHelper
+
+  scenario 'Application becomes inactive' do
+    given_i_am_signed_in
+    and_i_have_submitted_applications
+
+    when_one_of_my_applications_becomes_inactive
+    then_i_can_see_the_change_in_content
+    and_i_can_add_another_application
+  end
+
+  def given_i_am_signed_in
+    create_and_sign_in_candidate
+  end
+
+  def and_i_have_submitted_applications
+    current_candidate.application_forms.delete_all
```

Comment:
Why?

Reply:
Because using `create_and_sign_in_candidate` results in a pre-completed app with choices

Reply:
Are you sure?